node/assembler/batch_fetcher.go

```diff
@@ -227,12 +227,12 @@ func (br *BatchFetcher) GetBatch(batchID types.BatchID) (core.Batch, error) {
 		case <-ctx.Done():
 			cancelFunc()
 			return nil, fmt.Errorf("operation canceled")
-		case fetchedBatch := <-res:
+		case fb := <-res:
 			count++
-			if fetchedBatch.Shard() == batchID.Shard() && fetchedBatch.Primary() == batchID.Primary() && fetchedBatch.Seq() == batchID.Seq() && bytes.Equal(fetchedBatch.Digest(), batchID.Digest()) {
-				br.logger.Infof("Found batch %v", fetchedBatch)
+			if fb.Shard() == batchID.Shard() && fb.Primary() == batchID.Primary() && fb.Seq() == batchID.Seq() && bytes.Equal(fb.Digest(), batchID.Digest()) {
+				br.logger.Infof("Found batch <%d,%d,%d> with digest %s", fb.Shard(), fb.Primary(), fb.Seq(), hex.EncodeToString(fb.Digest()))
```

Comment:
Use this to print the batch ID so that we have uniform look and feel: `types.BatchIDToString(batchID)`